### PR TITLE
[PR-20586] documentation for cookies

### PR DESCRIPTION
**Pull request by necroua**
## Related Stories

[](https://talkable.atlassian.net/browse/PR-20586)

### CODE REVIEW

File: source/integration/custom/cookies.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
+.. _integration/custom/cookies:
+.. include:: /partials/common.rst
+
+.. meta::
+   :description: This is all about how we use cookies.
+
+Cookies
+####################
```
Comment (dgtv) [inline source/integration/custom/cookies.rst:8]:
```suggestion
#######
```

File: source/integration/custom/cookies.rst [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+   * - _ga
+     - Analytics and Performance
+     - talkable.com
+     - 400
+.. _multiple: https://newrelic.com/termsandconditions/cookie-policy/cookie-table
+
+.. _integration/custom/cookies/gdpr_and_ccpa_compliance:
+
+GDPR and CCPA compliance
+--------------------
```
Comment (dgtv) [inline source/integration/custom/cookies.rst:74]:
```suggestion
------------------------
```

File: source/integration/custom/cookies.rst [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Under GDPR and CCPA, setting such cookies does not require user consent.
+Post-consent (e.g., during signup), Talkable activates standard cookies to monitor user activities, aligning with regulatory guidelines.
+
+.. note::
+   Limitations of compliance mode seriously hinders Talkable ability to track referrals and conversions.
+   It is advised to use compliant mode only if you are legally obligated to do so.
+
+To enable GDPR and CCPA compliant mode, please contact your Customer Success Manager.
+
+
```
Comment (dgtv) [inline source/integration/custom/cookies.rst:87]:
```suggestion
```

**Review by dgtv — APPROVED** [review]
LGTM except rst fixes
@zhuravel Can you take a look too please? Thanks

File: source/integration/custom/cookies.rst [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+Cookies
+####################
+
+At Talkable, we prioritize enhancing your digital experience.
+Our use of cookies is instrumental in this endeavor, as they play a critical role in tracking referrals and conversions, personalizing content, and maintaining the overall performance and reliability of our services.
+
+Understanding the Cookies We Use
+--------------------------------
+We broadly categorize our cookies and similar technologies as follows:
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:15]:
```suggestion
--------------------------------

We broadly categorize our cookies and similar technologies as follows:
```

File: source/integration/custom/cookies.rst [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+.. list-table::
+   :widths: 30 30 30 10
+   :header-rows: 1
+
+   * - Cookie name
+     - Purpose
+     - Domain(s)
+     - Retention period(Days)
+   * - tkbl_session_id
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:42]:
```suggestion
   * - ``tkbl_session_id``
```

File: source/integration/custom/cookies.rst [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+     - Strictly Necessary
+     - | talkable.com
+       | shop domain
+     - 0
+   * - tkbl_session
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:47]:
```suggestion
   * - ``tkbl_session``
```

File: source/integration/custom/cookies.rst [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+     - Functionality
+     - | talkable.com
+       | shop domain
+     - 400
+   * - tkbl_cvuuid
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:52]:
```suggestion
   * - ``tkbl_cvuuid``
```

File: source/integration/custom/cookies.rst [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+     - Functionality
+     - | talkable.com
+       | shop domain
+     - 400
+   * - `multiple`_
+     - Analytics and Performance
+     - \*.newrelic.com
+     - 0-400
+   * - bugsnag*
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:61]:
```suggestion
   * - ``bugsnag*``
```

File: source/integration/custom/cookies.rst [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+     - Analytics and Performance
+     - \*.bugsnag.com
+     - 0-400
+   * - _ga
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:65]:
```suggestion
   * - ``_ga``
```

File: source/integration/custom/cookies.rst [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+     - Functionality
+     - | talkable.com
+       | shop domain
+     - 400
+   * - tkbl_cvuuid
+     - Functionality
+     - | talkable.com
+       | shop domain
+     - 400
+   * - `multiple`_
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:57]:
```suggestion
   * - `multiple <https://newrelic.com/termsandconditions/cookie-policy/cookie-table>`_
```

File: source/integration/custom/cookies.rst [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+     - 0-400
+   * - bugsnag*
+     - Analytics and Performance
+     - \*.bugsnag.com
+     - 0-400
+   * - _ga
+     - Analytics and Performance
+     - talkable.com
+     - 400
+.. _multiple: https://newrelic.com/termsandconditions/cookie-policy/cookie-table
```
Comment (zhuravel) [inline source/integration/custom/cookies.rst:69]:
```suggestion
     - 400
```

**Pull request merged**